controller/userController.js

```diff
@@ -13,11 +17,29 @@ const login = (req, res) => {
         if (user) {
             const saltedPassword = SHA256(req.body.password + user.salt).toString();
             if (saltedPassword === user.passwordHash) {
-                const token = jwt.sign({ id: user._id }, AppConfig.SECRET, {
-                    expiresIn: 7200 // expires in 2 hours
+                // Generate JWT
+                const token = jwt.sign({ id: user._id }, AppConfig.SECRET);
+                // const token = jwt.sign({ id: user._id }, AppConfig.SECRET, { expiresIn: AppConfig.TOKEN_LIFESPAN });
+                // const refreshToken = jwt.sign({ id: user._id }, AppConfig.REFRESH_TOKEN_SECRET, { expiresIn: AppConfig.REFRESH_TOKEN_LIFESPAN });
+
+                // Password & salt should be used only during login
+                user.passwordHash = null;
+                user.salt = null;
+
+                // Generate response
+                const response = {
+                    auth: true,
+                    status: "Logged in",
+                    user: user,
+                    token: token
+                };
+                return res.status(HttpStatus.OK).json(response)
+            } else {
+                res.send(HttpStatus.FORBIDDEN).json({
+                    success: false,
+                    message: 'Incorrect username or password'
                 });
-                return res.status(HttpStatus.OK).json({ auth: true, user, token });
-            };
+            }
         }
         return res.status(HttpStatus.NOT_FOUND).json({
             status: 'Error',
```

Comment:
In one case we are sending success and message properties, and in another status. Should be generalized to one or the other.